### ISSUE TITLE
Quartznet replicate some results

### ISSUE THREAD

**Issue by Miamoto** [NVIDIA/NeMo#3261]
Hello! 

I am trying to replicate Quartznet 5x5 results to get around 11/12% WER on the test set.
So far my last experiment was using 4 gpus, with batch 64, using speed perturbed data, specaugmentation, learning rate 0.01, dropout 0.2. I got around 23.18% on the validation and 17.11% on test set. Part of my config file is below:

<img width="702" alt="Captura de ecrã 2021-11-29, às 12 55 10" src="https://user-images.githubusercontent.com/15928244/143871857-85e7f9e4-f1f9-40f2-b9f7-0dff50e08d20.png">

Here I show my tensorboard results for training loss and validation loss: 
<img width="392" alt="Captura de ecrã 2021-11-29, às 12 49 48" src="https://user-images.githubusercontent.com/15928244/143871967-ffe62a03-7f10-448e-aaab-610f52291a79.png">
<img width="383" alt="Captura de ecrã 2021-11-29, às 12 49 52" src="https://user-images.githubusercontent.com/15928244/143871974-70da4f61-735c-46d2-94bf-05261a9f10f9.png">
<img width="376" alt="Captura de ecrã 2021-11-29, às 12 49 59" src="https://user-images.githubusercontent.com/15928244/143871990-e897e64e-77e2-44cb-94d5-3d47eeeee979.png">

<img width="428" alt="Captura de ecrã 2021-11-29, às 12 49 55" src="https://user-images.githubusercontent.com/15928244/143871984-cb3e0b91-130d-4f9a-8d1d-b073deb3bc27.png">

Do you guys have any guess what I should go for to reach a better WER for this architecture? Thank you, guys. Also, If I wanna to replicate results for architecture 5x3, using 4 gpus what hyperparameters should I use? 

Thanks a lot!

**Comment by titu1994** [comment]
Are you training on Librispeech ? The hyper parameters we have for Librispeech are trained for 400 epochs. 

From what I can see, your betas are incorrect (0.95, 0.5). Warmup steps should be 1000 but no warmup ratio. 

I personally haven't trained a QN 5x5 so I don't know what scores will be achieved. You could try citrinets (Citrinet 256) or Conformer (Conformer small) to train on Librispeech and get better accuracy

**Comment by Miamoto** [comment]
Hello! I am training with WSJ! Yeah, my goal is to try to get a baseline for architectures that do not use attention/memory mechanisms. So do you think that changing those things you mentioned are enough? Thanks a lot!

**Comment by titu1994** [comment]
If you need a baseline, use Citrinet or Conformer small on WSJ. Much better performance than QuartzNet and you can use the small variants (10~ M params) to get great scores

**Comment by titu1994** [comment]
The documentation page for ASR has a list of all the pretrained models. Each model after restore_from has it's config in model.cfg

**Comment by Miamoto** [comment]
Hey! If i load Citrinet 256 from stt_en_citrinet_256, there is already bpe tokens in there. Is it okay to use this setup directly for WSJ? And for training with 4 gpus do I still train the model with learning rate 0.05 and batch 32? Or do I change to batch 64 and learning rate 0.01? 

Thanks

**Comment by titu1994** [comment]
For finetuning the models you can reduce the LR to 0.01 or lower, batch size should be whatever is most efficient for training without oom.
